rasa/core/config.py

```diff
@@ -33,9 +45,108 @@ def load(config_file: Optional[Union[Text, Dict]]) -> List["Policy"]:
     return PolicyEnsemble.from_dict(config_data)
 
 
+def migrate_fallback_policies(config: Dict) -> Tuple[Dict, List["StoryStep"]]:
+    from rasa.core.policies.fallback import FallbackPolicy
+    from rasa.core.policies.two_stage_fallback import TwoStageFallbackPolicy
```

Comment:
I wanted to avoid importing the entire code base in case somebody uses the `load` function. Made it a top level import (it was very likely imported before anyway)